tsdb/engine/tsm1/engine_test.go

```diff
@@ -578,123 +578,100 @@ func TestEngine_CreateIterator_Condition(t *testing.T) {
 // Ensures that deleting series from TSM files with multiple fields removes all the
 /// series
 func TestEngine_DeleteSeries(t *testing.T) {
-	// Generate temporary file.
-	f, _ := ioutil.TempFile("", "tsm")
-	f.Close()
-	os.Remove(f.Name())
-	walPath := filepath.Join(f.Name(), "wal")
-	os.MkdirAll(walPath, 0777)
-	defer os.RemoveAll(f.Name())
-
-	// Create a few points.
-	p1 := MustParsePointString("cpu,host=A value=1.1 1000000000")
-	p2 := MustParsePointString("cpu,host=B value=1.2 2000000000")
-	p3 := MustParsePointString("cpu,host=A sum=1.3 3000000000")
+	for _, index := range tsdb.RegisteredIndexes() {
+		t.Run(index, func(t *testing.T) {
+			// Create a few points.
+			p1 := MustParsePointString("cpu,host=A value=1.1 1000000000")
+			p2 := MustParsePointString("cpu,host=B value=1.2 2000000000")
+			p3 := MustParsePointString("cpu,host=A sum=1.3 3000000000")
```

Comment:
I thought it was being used in a bunch of places but it seems to be less than I thought. Indentation was one concern. Another concern was DRY and consistency.